### DISCUSSION TITLE
OTG CR-10 smart compatible?

### DISCUSSION THREAD

**Discussion by thomasvjohansen** [feelfreelinux/octo4a#118] [Q&A]
Hi I have tried with two phones trying to connect to my printer Creality CR-10 smart.

both says printer not connected. im using a OTG y-cable. 

Anyone  have success with CR-10 smart?

**Comment by happy1234567** [comment]
Did you disconnect the and remove the external USB cable and throw it away  and re-route the two connections on the boards  inside the printer? You have to disable their cloud system and use a micro USB cable from the internal board to your phone and it will work.  There are You tube videos on how to do this.

**Reply by thomasvjohansen** [reply]
yeah I found out :-) In my wildest fantasy I haven't thought the usb only worked for a creality usb cam. Opened it up and switched cable and attached a usb cable. But my experience afterwards was that I had to connect everything in a special order to get it work. Like when printer is shutdowned, connect phone and printer to OTG cable, then connect to usb charger and then turn on printer. And then even though the phone charges, it uses more power faster than charging. 
I ended up in deciding to buy a rpi zero 2w and go from there.
